.github/ISSUE_TEMPLATE/product_update_facdb.md

```diff
@@ -0,0 +1,146 @@
+---
+ name: Product Update: FacDB
+ about: Product Update: FacDB
+ title: "{{ env.VERSION }} FacDB Update"
+ labels: 'data update'
+---
+
+# FacDB Source Data Updates
+
+Like most of our data products, source data must be updated in data library before FacDB is run. As there are are many source datasets with varied update processes, this issue template should be opened to track progress towards updating all source data
+
+**All source data listed is to be uploaded as .csv files**
+
+## Geosupport
+
+- [ ] Ensure latest Geosupport version is present in production EDM Docker images
+
+## Version Env
+
+- [ ] Update previous version for QCQA in `version.env` file
```

Comment:
This isn't used either anymore, is it?

Reply:
it was used in the last build

Reply:
Gotcha. I don't think it's used or needed. There's some fairly hacky code in `plan` that sets `VERSION_PREV` in the (python) environment by default, and all builds github workflows have a command to export the recipe's logged environment to the bash environment for running the build

Reply:
should I remove this section then? Or should I have a note to try remove the version file and try building it using `plan` during our next build

Reply:
See [here](https://github.com/NYCPlanning/data-engineering/blob/main/dcpy/lifecycle/builds/plan.py#L65) and [here](https://github.com/NYCPlanning/data-engineering/blob/main/.github/workflows/facilities_build.yml#L62)

Reply:
awesome, thanks @fvankrieken